### DISCUSSION TITLE
Error when make entity. Symfony 4.4

### DISCUSSION THREAD

**Discussion by merrily96** [symfony/symfony#47045] [Q&A] [answered]
when i run 
` php bin/console make:entity`
the output
`[critical] Error thrown while running command ""make:entity"". Message: "Command "make:entity" is not defined.

Did you mean one of these?
    make:admin:crud
    make:admin:dashboard
    make:admin:migration"`

help me to solve this

**Comment by ahundiak** [comment, marked as answer]
As mention on your stackoverflow question you need to install the maker bundle with `composer require --dev symfony/maker-bundle`

You should be aware that, for some reason, 4.4 has basically been abandoned after the 4.4.41 version.  The Doctrine orm 2.12 package breaks the Symfony/Doctrine interface.  You will, among other things, see a strange annotation error as soon as you try running `make:entity`.  You can seemingly workaround the various problems by setting orm back to 2.11 but that is potentially dangerous.  So avoid going past 4.4.41 if possible.
